.gitignore

```diff
@@ -1,8 +1,8 @@
+.venv
+.git
+__pycashe__
+*.pyc
+*.pyo
+.DS_Store
 .idea/
-.vscode/
-*.iml
 .env
```

Comment:
Consider adding `.env` to the `.gitignore` file to prevent sensitive information from being committed to the repository.